vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue

```diff
@@ -9,11 +9,22 @@
       color="#3cba54">
       <v-icon >mdi-plus</v-icon>
     </v-btn>
+
+    <v-alert id="form-alert"
+    v-model="alert"
+    transition="scroll-y-transition"
+    :type="alertType"
+    elevation="10"
+    dismissible
+    absolute> 
```

Comment:
Remove absolute here since we moved it to CSS

Reply:
will do!